core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -719,33 +723,46 @@ object JsonCodec {
   )(x => (x.publisher, x.offer, x.sku, x.version))
 
   implicit val landingZoneResourcesDecoder: Decoder[LandingZoneResources] =
-    Decoder.forProduct7("clusterName",
-                        "batchAccountName",
-                        "relayNamespace",
-                        "storageAccountName",
-                        "vnetName",
-                        "batchNodesSubnetName",
-                        "aksSubnetName"
+    Decoder.forProduct11(
+      "clusterName",
+      "batchAccountName",
+      "relayNamespace",
+      "storageAccountName",
+      "vnetName",
+      "postgresName",
+      "logAnalyticsWorkspaceName",
+      "batchNodesSubnetName",
+      "aksSubnetName",
+      "postgresSubnetName",
+      "computeSubnetName"
     )(
       LandingZoneResources.apply
     )
 
-  implicit val landingZoneResourceEncoder: Encoder[LandingZoneResources] = Encoder.forProduct7(
+  implicit val landingZoneResourceEncoder: Encoder[LandingZoneResources] = Encoder.forProduct11(
```

Comment:
Adding a few more things to `LandingZoneResources` since I think we'll need them soon/later.